kitsune/sumo/static/sumo/css/print.css

```diff
@@ -5,14 +5,52 @@ body {
     margin: 0;
     padding: 0;
 }
+
+body.scroll-header {
+    padding-top: 0;
+}
+
+body > header {
+    padding-bottom: 5px;
+    position: relative;
+}
+
+body > header a {
+    border-bottom: 0;
+}
+
+body.html-ltr .container_12 {
+    width: auto;
```

Comment:
@rehandalal can you help clarify how you would want this ordered? I'm guessing:

body.html-ltr .container_12
body.scroll-header
body > header
body > header a

Not sure if classes on a tag would come before or after a tag with a child selector?  I'm not a huge fan of alphabetizing CSS...sometimes I think an alphabetic chronology obscures semantics, but I will do my best :)